AcadeMeData/migrations/0005_messageTags_test_data.py

```diff
@@ -9,7 +9,7 @@ class Migration(migrations.Migration):
 
     def generate_msgtagsdata(apps, schema_editor):
         from AcadeMeData.models import User, Messages, MessageTags
-        userrnd1 = User.create_user('u', 'u@gmail.com', 'pas', 'S', 'BS', 'CS')
+        userrnd1 = User.create_user('u', 'u@gmail.com', 'pas', 'BS', 'CS')
         msgID = Messages.create_message(1, userrnd1, 'Hello this is test message')
```

Comment:
This file has been modified to be compatible with the latest User model
see that the field Type is no longer in the user model so I have deleted it from the test data files 
('S')